docs/reference/builder.md

```diff
@@ -493,6 +494,25 @@ before each new `FROM` command.
 assumes a `latest` by default. The builder returns an error if it cannot match
 the `tag` value.
 
+<<<<<<< HEAD
+=======
+## MAINTAINER
+
+    MAINTAINER <name>
+
+The `MAINTAINER` instruction allows you to set the *Author* field of the
+generated images.
+
+## INCLUDE
+
+    INCLUDE path/to/file
+
+The `INCLUDE` instruction tells the `Dockerfile` parser to read in the
+specified file and process its contents before continuing with the current
+file. All commands in the referenced file will be processed as if they
+were included as part of the current file.
+
+>>>>>>> 3618080... Support INCLUDE in Dockerfiles
```

Comment:
Oh! Shouldn't @GordonTheTurtle catch those? I think there was something to catch them?